src/amo/components/AddonMoreInfo/index.js

```diff
@@ -65,7 +67,9 @@ export class AddonMoreInfoBase extends React.Component {
           {i18n.gettext('Visit stats dashboard')}
         </Link>
       ) : null,
-      version: addon.current_version.version,
+      // Themes are sent with a version of "0", which we ignore.
+      version: addon.current_version.version !== '0' ?
```

Comment:
This needs to check `addon.type` similar to how `addonHasVersionHistory()` works.

I was curious if a real extension could have a version of "0" and indeed it can:

<img width="735" alt="screenshot 2017-10-02 09 50 34" src="https://user-images.githubusercontent.com/55398/31083425-465c4896-a757-11e7-8c6e-50df80a80617.png">

Reply:
Hah! Good catch, I didn't think that'd be allowed.